11:55:07.922 INFO Test execution started: second_test
11:55:07.923 INFO Browser: chrome-remote
11:55:07.924 INFO Environment: test_env
11:55:07.924 INFO Using data:
    order_type: Plats
    document_type: LETTER (PL)
    num_of_pages: 1
    num_of_lots: 1
    section: GRANTOR
    first_name: First Name
    alert_text: Please Enter Account OR Email OR Name
    url: http://{}:{}@10.0.1.73:8001/69999/?ReturnUrl=https://crs.qa.kofile.com/69999/Account/ReturnClaim&amp;Referrer=
11:55:08.581 ERROR JSONDecodeError: Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "c:\users\alexandr.may\appdata\local\programs\python\python37-32\lib\site-packages\golem\test_runner\test_runner.py", line 173, in run_test
    self.test_module.test(execution.data)
  File "C:\Users\alexandr.may\testdir\projects\some_project\tests\second_test.py", line 35, in test
    dataInfile = json.loads(file2.read())
  File "c:\users\alexandr.may\appdata\local\programs\python\python37-32\lib\json\__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "c:\users\alexandr.may\appdata\local\programs\python\python37-32\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "c:\users\alexandr.may\appdata\local\programs\python\python37-32\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

11:55:08.754 INFO Test Result: CODE ERROR
